.github/workflows/release.yml

```diff
@@ -37,12 +37,14 @@ jobs:
       # IMPORTANT: this permission is mandatory for trusted publishing to pypi
       id-token: write
     needs: release-please
-    if: ${{ needs.release-please.outputs.release_created }}
-    container:
-      image: "python:3.13"
+    if: ${{ fromJSON(needs.release-please.outputs.release_created || false) }}
```

Comment:
Unrelated to the change but future proofs us if/when we upgrade to Release Please v4